lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -21,146 +22,131 @@ describe('NxRadio', function() {
+  const renderEl = rtlRenderElement<Props>(NxRadio, minimalProps);
+  const quickRender = rtlRender<Props>(NxRadio, minimalProps);
 
   it('renders a <label> containing a radio <input> and .nx-radio__circle and .nx-radio__outer-circle  elements',
       function() {
-        const shallowRender = getShallow();
+        const el = renderEl()!;
 
-        expect(shallowRender).toMatchSelector('label.nx-radio');
+        expect(el.tagName).toBe('LABEL');
+        expect(el).toHaveClass('nx-radio');
 
-        expect(shallowRender.find('input')).toHaveProp('name', 'color');
-        expect(shallowRender.find('input')).toHaveProp('type', 'radio');
-        expect(shallowRender.find('input')).toHaveProp('disabled', false);
-        expect(shallowRender.find('input')).toHaveProp('checked', false);
+        expect(el.querySelector('input')).toHaveAttribute('name', 'color');
+        expect(el.querySelector('input')).toHaveAttribute('type', 'radio');
+        expect(el.querySelector('input')).not.toHaveAttribute('disabled');
+        expect(el.querySelector('input')).not.toHaveAttribute('checked');
 
-        expect(shallowRender.find('input')).toHaveClassName('nx-radio__input');
+        expect(el.querySelector('input')).toHaveClass('nx-radio__input');
 
-        expect(shallowRender.find('svg.nx-radio__circle')).toExist();
-        expect(shallowRender.find('.nx-radio__circle .nx-radio__outer-circle')).toExist();
+        expect(el.querySelector('svg')).toHaveClass('nx-radio__circle');
+        expect(el.querySelector('circle')).toHaveClass('nx-radio__outer-circle');
       }
   );
 
   it('adds classes specified with the className prop', function() {
-    const component = getShallow({ className: 'foo' });
+    const component = renderEl({ className: 'foo' });
 
-    expect(component).toHaveClassName('foo');
-    expect(component).toHaveClassName('nx-radio');
+    expect(component).toHaveClass('foo');
+    expect(component).toHaveClass('nx-radio');
   });
 
-  it('renders .nx-radio__inner-circle iff it is checked', function() {
-    expect(getShallow()).not.toContainMatchingElement('.nx-radio__inner-circle');
-    expect(getShallow({ isChecked: true })).toContainMatchingElement('.nx-radio__circle .nx-radio__inner-circle');
-    expect(getShallow({ isChecked: true, disabled: true }))
-        .toContainMatchingElement('.nx-radio__circle .nx-radio__inner-circle');
+  it('renders .nx-radio__inner-circle if it is checked', function() {
+    expect(renderEl()?.querySelector('circle')).not.toHaveClass('nx-radio__inner-circle');
+
+    expect(renderEl({ isChecked: true })?.querySelector('circle')).toHaveClass('nx-radio__inner-circle');
+    expect(renderEl({ isChecked: true, disabled: true })?.querySelector('circle'))
+        .toHaveClass('nx-radio__inner-circle');
   });
 
-  it('uses null as the value passed to onChange if the supplied value is null', function() {
+  it('uses null as the value passed to onChange', async function() {
```

Comment:
Why'd you get rid of the second half of the test name here? It's important

Reply:
> Why'd you get rid of the second half of the test name here? It's important

Don't know why it was changed, must've done it accidentally: bfcfcb45e3e795890140811d7307ef4fb3153403